packages/tools/client-debugger/client-debugger-view/src/components/FluidTreeView.tsx

```diff
@@ -0,0 +1,38 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import React from "react";
+import { HasContainerId, FluidObjectTreeNode } from "@fluid-tools/client-debugger";
+import { Accordion } from "./utility-components/";
+import { TreeDataView } from "./TreeDataView";
+
+/**
+ * {@link TreeView} input props.
+ */
+export interface FluidTreeViewProps extends HasContainerId {
+	node: FluidObjectTreeNode;
+}
+
+/**
+ * Displays visual summary trees for DDS_s within the container.
+ */
+export function FluidTreeView(props: FluidTreeViewProps): React.ReactElement {
+	const { containerId, node } = props;
+	// Accordion header:  fluidObjectId, metadata, typeMetadata
+	// Accordion children: (all of the rendered TreeDataView nodes)
+	return (
+		<>
```

Comment:
Wrapped the `Accordion` and the map function in a React Fragment which allows the `TreeDataView` components to be rendered as siblings of the `Accordion` component, rather than being its children. 

If not, the values are not displayed in the `ValueView` component which fails the unit test.